Tests/Parser/Client/fixtures/library.yml

```diff
@@ -689,3 +689,21 @@
     type: library
     name: IPinfo
     version: 3.0.0
+-
+  user_agent: Android.LVLDM
+  client:
+    type: library
+    name: Android License Verification Library
+    version: ""
+-
+  user_agent: Mozilla/5.0 (Unknown; Linux x86_64) AppleWebKit/538.1 (KHTML, like Gecko) PhantomJS/2.1.1 Safari/538.1
+  client:
+    type: library
```

Comment:
should not this be a browser? project considered itself one, at least https://github.com/ariya/phantomjs

Reply:
It should, but I also saw that `SlimerJS` is also a library, instead of browser.